generators/python-v2/codegen/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@fern-api/pythonv2-codegen",
+  "name": "@fern-api/python-v2-codegen",
```

Comment:
The directory has a dash, but the package name didn't. Not sure which you prefer